bundle/manifests/splunk-operator.clusterserviceversion.yaml

```diff
@@ -915,5 +915,5 @@ spec:
   relatedImages:
   - image: docker.io/splunk/splunk:9.0.3-a2
     name: splunk-enterprise
-  replaces: splunk-operator.v2.1.0
-  version: 2.2.0
+  replaces: splunk-operator.v2.2.0
```

Comment:
Can you check if this is correct?

Reply:
we are good we do not use it directly , we need to create separate one for openshift and operatorhub.